File: Sistema/src/app/models/SessaoLista.Model.ts
export class SessaoListaModel {
  SesCodi: number;
  SesDesc: string;
  SesDtHr: Date;
  SesLibe: boolean;
  SesStat: boolean;
  LojCodi: number;
  GraCodi: number;
  GraNome: string;
  TiSCodi: number;
  TiSNome: string;
  SesNume: number;
  SesNome: string;
  SesAgap: boolean;
  SesVlAg: number;

  constructor(
    SesCodi: number,
    SesDesc: string,
    SesDtHr: Date,
    SesLibe: boolean,
    SesStat: boolean,
    LojCodi: number,
    GraCodi: number,
    GraNome: string,
    TiSCodi: number,
    TiSNome: string,
    SesNume: number,
    SesNome: string,
    SesAgap: boolean,
    SesVlAg: number
  ) {
    this.SesCodi = SesCodi;
    this.SesDesc = SesDesc;
    this.SesDtHr = SesDtHr;
    this.SesLibe = SesLibe;
    this.SesStat = SesStat;
    this.LojCodi = LojCodi;
    this.GraCodi = GraCodi;
    this.GraNome = GraNome;
    this.TiSCodi = TiSCodi;
    this.TiSNome = TiSNome;
    this.SesNume = SesNume;
    this.SesNome = SesNome;
    this.SesAgap = SesAgap;
    this.SesVlAg = SesVlAg;
  }
}

File: API/Models/SessaoListaModel.cs
namespace API_Visitatus.Models
{
    public class SessaoListaModel
    {
        public long SesCodi { get; set; }
        public string? SesDesc { get; set; }
        public DateTime SesDtHr { get; set; }
        public bool SesLibe { get; set; }
        public bool SesStat { get; set; }
        public long LojCodi { get; set; }
        public int GraCodi { get; set; }
        public string? GraNome { get; set; }
        public int TiSCodi { get; set; }
        public string? TiSNome { get; set; }
        public long SesNume { get; set; }
        public string? SesNome { get; set; }
    }
}
